ios/Classes/ReactiveBle/CentralManagerDelegate.swift

```diff
@@ -9,7 +9,7 @@ enum ConnectionChange {
 final class CentralManagerDelegate: NSObject, CBCentralManagerDelegate {
 
     typealias StateChangeHandler = (CBManagerState) -> Void
-    typealias DiscoveryHandler = (CBPeripheral, ServiceData, RSSI) -> Void
+    typealias DiscoveryHandler = (CBPeripheral, [String : Any], RSSI) -> Void
```

Comment:
If the type-alias `AdvertisementData` is introduced, please use it here.

Reply:
Done.

Reply:
Ehm. Did you push it?

Reply:
OK, sorry about that. Missed that there were 2 DiscoveryHandler.
Also, I reintroduced ServiceData that was actually used in PluginController